packages/core/botpress/yarn.lock

```diff
@@ -7702,6 +7702,10 @@ trim-right@^1.0.1:
   version "1.0.1"
   resolved "https://registry.yarnpkg.com/trim-right/-/trim-right-1.0.1.tgz#cb2e1203067e0c8de1f614094b9fe45704ea6003"
 
+triple-beam@^1.3.0:
+  version "1.3.0"
+  resolved "https://registry.yarnpkg.com/triple-beam/-/triple-beam-1.3.0.tgz#a595214c7298db8339eeeee083e4d10bd8cb8dd9"
+
```

Comment:
@emirotin , was it added on purpose or just left from adding/removing some packages?

Reply:
Hm I thought I have cleaned it up. Will fix thanks